sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/query/PartTreeReactiveCosmosQuery.java

```diff
@@ -41,7 +40,7 @@ protected DocumentQuery createQuery(ReactiveCosmosParameterAccessor accessor) {
         final DocumentQuery query = creator.createQuery();
 
         if (tree.isLimiting()) {
```

Comment:
thank you for using UnsupportedOperationException rather than NotImplementedException as this reproduces our dependency on commons.

Reply:
Yes, that was my exact intention behind this change. :)